src/Sfa.Tl.Matching.Application.IntegrationTests/BankHoliday/CalendarHttpClient.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.IO;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Reflection;
+
+namespace Sfa.Tl.Matching.Application.IntegrationTests.BankHoliday
+{
```

Comment:
Raneme this to Fake OR Test CalendarHttpClient

Reply:
Renamed in new branch. Also renamed PostcodesIoHttpClient